R/zchunk_LA100.GTAP_downscale_ctry.R

```diff
@@ -12,9 +12,9 @@
 #' @importFrom assertthat assert_that
```

Comment:
Do you fill in a `@details` field? If not, I think probably would be good to do.